specification/keyvault/data-plane/readme.md

```diff
@@ -429,4 +429,11 @@ directive:
     from: securitydomain.json
     where: $.definitions.TransferKey.properties.key_format
     reason: Consistency with other properties
+  - suppress: DOUBLE_FORWARD_SLASHES_IN_URL
+    from: rbac.json
```

Comment:
Still, since both of these problems happen in other files, does suppression work if you get rid of `from: rbac.json`? Is it required?